### ISSUE TITLE
no scheme detected upon setting the colorscheme value to "jb"

### ISSUE THREAD

**Issue by effinsky** [devsjc/vim-jb#1]
this is in:

```
NVIM v0.9.4
Build type: Release
LuaJIT 2.1.1700008891
```
really want to try this out, though

**Comment by effinsky** [comment]
yes, of course, I downloaded it with Packer, as it happens. Sorry, in my particular $VIMRUNTIME echoes nothing.  I have found the colors manually, but this folder stores the builtin schemes only, looks like:
<img width="1181" alt="Screenshot 2023-12-01 at 18 09 27" src="https://github.com/devsjc/vim-jb/assets/54535543/0514aee0-ef2f-4e1f-80bf-0224c11b8543">

**Comment by effinsky** [comment]
yes, of course, I downloaded it with Packer, as it happens. Sorry, in my particular $VIMRUNTIME echoes nothing.  I have found the colors manually, but this folder stores the builtin schemes only, looks like:
<img width="1181" alt="Screenshot 2023-12-01 at 18 09 27" src="https://github.com/devsjc/vim-jb/assets/54535543/0514aee0-ef2f-4e1f-80bf-0224c11b8543">

the packer plugin manager folder elsewhere stores the plugins and here we find your plugin installed:
<img width="329" alt="Screenshot 2023-12-01 at 18 12 31" src="https://github.com/devsjc/vim-jb/assets/54535543/be68bbeb-7253-4feb-808e-523b1752d36c">

but trying to activate it results in defaulting to the builtin fallback:
<img width="849" alt="Screenshot 2023-12-01 at 18 15 44" src="https://github.com/devsjc/vim-jb/assets/54535543/04bebdda-00b9-45c3-82ec-cd210be28846">

**Comment by devsjc** [comment]
I am not sure I'm able to replicate this. On a fresh install of Neovim on Ubuntu (v0.6.1)...

1. I cloned packer onto my packpath:

```sh
$ git clone --depth 1 https://github.com/wbthomason/packer.nvim ~/.local/share/nvim/site/pack/packer/start/packer.nvim
```

2. Created a `plugins.lua` file in `~/.config/nvim/lua/plugins.lua` with the following contents:

```lua
return require('packer').startup(function()
  -- Packer can manage itself
  use 'wbthomason/packer.nvim'
  use 'devsjc/vim-jb'
end)
```

3. Created an `init.vim` in `~/.config/nvim` with just the following contents:

```vim
lua require('plugins')
```

4. Opened `nvim` and ran `:PackerSync`. The colorscheme `jb` was then available in the tab completed colorscheme options:

![image](https://github.com/devsjc/vim-jb/assets/47188100/0ec84d2c-6eb9-4799-9708-de8f14f05a1d)

**Comment by fipachu** [comment]
> <img width="902" alt="Screenshot 2023-12-04 at 12 51 38" src="https://github.com/devsjc/vim-jb/assets/54535543/535c530f-f245-435e-94e8-ee04b26ece4a">

I've had the same issue. I installed vim-plug, installed vim-jb, set colorscheme to jb and got a bunch of errors.

Then I did the same steps in vim and everything worked as it should.

**Comment by effinsky** [comment]
so it's just not neovim-compatible, then?

**Comment by devsjc** [comment]
Have you tried running `:PackerSync` to get the latest version? Again I can't replicate this as it sets the theme for me, same version as specified above:

<img width="719" alt="image" src="https://github.com/devsjc/vim-jb/assets/47188100/ea2d8c88-12ff-4677-a16a-042659cafb8f">

But yes, you are right in that I have built the scheme for *vim*, not nvim - so whilst it is nvim *compatible* it isn't nvim *optimised*; the colors do look a little different than I'd expect, but perhaps I haven't set truecolor properly on the nvim or something.

I'd welcome a PR adding proper nvim support if you know what to do for that!

**Comment by effinsky** [comment]
of course. actually my packer syncs automatically on save. to be precise, however, I don't think it's nvim-compatible at all. although I'd like to be proven wrong by someone actually using it successfully with nvim.

beyond this, I think to make it really nvim-usable it'd have to be rewritten in lua for convenience.

**Comment by devsjc** [comment]
I've updated to remove the `ctermul` line as it wasn't being used anyway, and it appears to be the source of some of the errors screenshotted above, so that error shuoldn't be present with the latest version.

It does seem to be at least minimally compatible with neovim: if I investigate the highlight groups in nvim via 

```
:so $VIMRUNTIME/syntax/hitest.vim
```

The `nvim`-prefixed groups are within seemingly relevant highlight groups specific to this scheme:

![Screenshot 2023-12-05 at 15 35 14](https://github.com/devsjc/vim-jb/assets/47188100/d7f279ac-02a5-4e35-8466-044ea27559fa)

I'm afraid a rewrite in lua is something that is beyond me, but in lieu of that I've added (https://github.com/devsjc/vim-jb/commit/b85e23cbfce2a34c961c238b799b12b08af7e10d) a block of nvim-specific highlight groups which should go a bit further to making it a useable scheme on neovim.

**Comment by effinsky** [comment]
thanks -- I can confirm that the theme loads now without errors.

**Comment by devsjc** [comment]
Thanks for raising the issue! Glad to hear it's working.
